### PR TITLE
en: add notices about import/scale with TiUP

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

- add notices about import/scale with TiUP

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/321
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/maintain-dm-using-tiup.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ tiup dm scale-in prod-cluster -N 172.16.5.140:8262
 
 The scale-out operation has an inner logic similar to that of deployment: the TiUP DM component first ensures the SSH connection of the node, creates the required directories on the target node, then executes the deployment operation, and starts the node service.
 
-For example, to scale out a DM-worker node in the `prod-cluster` cluster, take the following steps:
+For example, to scale out a DM-worker node in the `prod-cluster` cluster, take the following steps. You can scale in the DM-master using similar methods:
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:140]:
```suggestion
For example, to scale out a DM-worker node in the `prod-cluster` cluster, take the following steps (scaling out DM-master has similar steps):
```

File: en/maintain-dm-using-tiup.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ tiup dm scale-in <cluster-name> -N <node-id>
 
 To use this command, you need to specify at least two arguments: the cluster name and the node ID. The node ID can be obtained by using the `tiup dm display` command in the previous section.
 
-For example, to scale in the DM-worker node on `172.16.5.140`, run the following command:
+For example, to scale in the DM-worker node on `172.16.5.140`, run the following command. You can scale in the DM-master using similar methods:
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:128]:
```suggestion
For example, to scale in the DM-worker node on `172.16.5.140` (similar to scaling in DM-master), run the following command:
```

File: en/maintain-dm-using-tiup.md [hunk 3]
```diff
@@ -249,8 +249,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 >
 > - TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
 > - You need to stop the original cluster before importing. 
+> - TiUP only supports importing to v2.0.0-rc.2 or later versions.
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:252]:
```suggestion
> - TiUP only supports importing to a DM cluster of v2.0.0-rc.2 or a later version.
```

File: en/maintain-dm-using-tiup.md [hunk 4]
```diff
@@ -249,8 +249,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
...
 > - The deployment directories of some components are different from those of the original cluster. You can execute the `display` command to view the details. 
 > - Run `tiup update --self && tiup update dm` before importing to make sure that the TiUP DM component is the latest version.
+> - Only one DM-master node is in the cluster after importing. Refer to [Scale out a cluster](#scale-out-a-cluster) to expand the DM-master.
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:255]:
```suggestion
> - Only one DM-master node exists in the cluster after importing. Refer to [Scale out a cluster](#scale-out-a-cluster) to scale out the DM-master.
```

**Review by yikeke — COMMENTED** [review]
Rest LGTM

**Review by lance6716 — COMMENTED** [review]
LGTM

**Comment by lance6716** [comment]
only translated part of Chinese PR?

**Comment by ireneontheway** [comment]
> only translated part of Chinese PR?

Some mirror mistakes only appear in Chinese PRs. So there is no need to align with other parts. @lance6716

**Review by lance6716 — APPROVED** [review]
Oh I have permission to Approve

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**